### PR TITLE
Fix for WFCORE-6656, CLI, shaded jar can't start embedded when logging json formater configured

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/WFCORE-6656

### CODE REVIEW

**Comment by jfdenise** [comment]
The full integration tests will fail until https://github.com/wildfly/wildfly/pull/17516 is nerged.

File: cli/pom.xml [hunk 1]
```diff
@@ -729,19 +729,11 @@
             </exclusions>
         </dependency>
 
+        <!-- Dependency needed by the CLI fat jar (client classifier) to support JBoss Logging JSON Formatter 
+             that still depends on EE8 javax.json API -->
```
Comment (jamezp) [inline cli/pom.xml:733]: Is the log manager required by CLI?

Comment (jamezp) [reply cli/pom.xml:733]: Sorry, it is. I remember it's needed for the embedded stuff now.

File: cli/pom.xml [hunk 2]
```diff
@@ -729,19 +729,11 @@
             </exclusions>
         </dependency>
 
+        <!-- Dependency needed by the CLI fat jar (client classifier) to support JBoss Logging JSON Formatter
```
Comment (jamezp) [inline cli/pom.xml:732]: Minor nit, it's not JBoss Logging that requires it, but the JBoss Log Manager.

Comment (jfdenise) [reply cli/pom.xml]: Thank-you. Fixed

**Review by jamezp — APPROVED** [review]

**Comment by yersan** [comment]
Thanks @jfdenise @jamezp

**Pull request merged**